README.md

```diff
@@ -120,7 +169,7 @@ nothing to commit, working directory clean
 3. Once your changes have been added and staged, you can make a commit with the `git commit` command. **To capture all outstanding changes in a commit, type `git commit -am "Your commit message"`, where `-a` refers to 'all changes' and `-m` (combined as `-am`) assigns a commit message of `"Your commit message"`.**
```

Comment:
This is inherited, but this conclusion is not compatible with [the process](https://flatiron-curriculum.gitbook.io/standards/authoring-process). It doesn't recall the objectives and it puts in way too much new knowledge that should have been explained in the body.